pycaret/internal/plots/time_series.py

```diff
@@ -23,8 +23,8 @@
     qq_subplot,
     time_series_subplot,
 )
-from pycaret.utils import _resolve_dict_keys
 from pycaret.utils.time_series import get_diffs
+from pycaret.utils.utils import _resolve_dict_keys
```

Comment:
Same comment here. Should we just use `pycaret.utils` instead of `pycaret.utils.utils`